[2023-11-20T06:17:11.642+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_python_operator.docker_command_sleep manual__2023-11-20T06:17:09.659452+00:00 [queued]>
[2023-11-20T06:17:11.650+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_python_operator.docker_command_sleep manual__2023-11-20T06:17:09.659452+00:00 [queued]>
[2023-11-20T06:17:11.650+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-20T06:17:11.668+0000] {taskinstance.py:1382} INFO - Executing <Task(DockerOperator): docker_command_sleep> on 2023-11-20 06:17:09.659452+00:00
[2023-11-20T06:17:11.677+0000] {standard_task_runner.py:57} INFO - Started process 70 to run task
[2023-11-20T06:17:11.680+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'example_python_operator', 'docker_command_sleep', 'manual__2023-11-20T06:17:09.659452+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/first.py', '--cfg-path', '/tmp/tmpce3kxw1m']
[2023-11-20T06:17:11.681+0000] {standard_task_runner.py:85} INFO - Job 36: Subtask docker_command_sleep
[2023-11-20T06:17:11.742+0000] {task_command.py:416} INFO - Running <TaskInstance: example_python_operator.docker_command_sleep manual__2023-11-20T06:17:09.659452+00:00 [running]> on host 49ac4921b293
[2023-11-20T06:17:11.928+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_python_operator' AIRFLOW_CTX_TASK_ID='docker_command_sleep' AIRFLOW_CTX_EXECUTION_DATE='2023-11-20T06:17:09.659452+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-20T06:17:09.659452+00:00'
[2023-11-20T06:17:12.499+0000] {docker.py:343} INFO - Starting docker container from image python:3.8.0
[2023-11-20T06:17:12.502+0000] {docker.py:351} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2023-11-20T06:17:15.119+0000] {docker.py:413} INFO - hello
[2023-11-20T06:17:19.121+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_python_operator, task_id=docker_command_sleep, execution_date=20231120T061709, start_date=20231120T061711, end_date=20231120T061719
[2023-11-20T06:17:19.228+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-20T06:17:19.250+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
